tests/Core/Config/SniffsExcludeArgsTest.php

```diff
@@ -144,15 +189,50 @@ public static function dataValidSniffs()
         $data      = [];
 
         foreach ($arguments as $argument) {
+            $data[$argument.'; empty string']     = [
+                'argument' => $argument,
+                'value'    => '',
+                'result'   => [],
+            ];
             $data[$argument.'; one valid sniff']  = [
                 'argument' => $argument,
                 'value'    => 'Standard.Category.Sniff',
+                'result'   => ['Standard.Category.Sniff'],
             ];
             $data[$argument.'; two valid sniffs'] = [
                 'argument' => $argument,
                 'value'    => 'StandardOne.Category.Sniff,StandardTwo.Category.Sniff',
+                'result'   => [
+                    'StandardOne.Category.Sniff',
+                    'StandardTwo.Category.Sniff',
+                ],
             ];
-        }
```

Comment:
Should a test be added with a duplicate sniff code being passed ?
And maybe tests with variations in case ? (both for valid as well as invalid)